test/metabase/search/scoring_test.clj

```diff
@@ -226,6 +230,29 @@
                     (sort-by score)
                     (map :id))))))))
 
+(defn search-results
+  "Returns search results as they are returned from the search query before scoring"
+  [search-ctx]
+  (mt/with-current-user (mt/user->id :crowberto)
+    (let [search-ctx (merge search.filter-test/default-search-ctx search-ctx)]
```

Comment:
We could use a `@#'` reference here an keep `default-search-ctx` private if we want. Not sure what our style guide for "visible for tests only" is.

Reply:
Anything in a `/test` namespace should only be visible for tests.